docs/domains/Domain.md

```diff
@@ -77,6 +77,7 @@ An element representing a cluster in the domain configuration.
 
 | Name | Type | Description |
 | --- | --- | --- |
+| `allowReplicasBelowMinDynClusterSize` | Boolean | If true (the default), the number of replicas is allowed to drop below the minimum dynamic cluster size configured in the WebLogic domain home configuration. Otherwise, the operator will ensure that the number of replicas is not below the minimum dynamic cluster. This setting applies to dynamic clusters only. |
```

Comment:
If true (the default), the number -> If true (the default), then the number
that the number of replicas is not below the minimum dynamic cluster -> that the number of replicas is not less than the minimum dynamic cluster setting.

Reply:
Thanks for the comment. I have updated the description to:

If true (the default), then the number of replicas is allowed to drop below the minimum dynamic cluster size configured in the WebLogic domain home configuration. Otherwise, the operator will ensure that the number of replicas is not less than the minimum dynamic cluster setting. This setting applies to dynamic clusters only.